### PR TITLE
fix: trainer for non trainable model

### PR DESCRIPTION
**Pull request by YuriCat**
This is just a bug, and we don't need any Batcher if self.optimizer is `None`.

### CODE REVIEW

File: handyrl/train.py [hunk 1]
```diff
@@ -349,8 +349,8 @@ def shutdown(self):
 
     def train(self):
         if self.optimizer is None:  # non-parametric model
-            print()
-            return
+            time.sleep(0.1)
```
Comment (ikki407) [inline handyrl/train.py:352]: Why is this needed?

Comment (YuriCat) [reply handyrl/train.py:352]: This is because without time.sleep(), train() will be called incessantly.
It may be better to prevent calling this function.

Comment (YuriCat) [reply handyrl/train.py:352]: The problem is that epoch will not proceed until train() is called.

**Comment by ikki407** [comment]
LGTM
